src/CheckBoxGroup/CheckBox.tsx

```diff
@@ -1,83 +0,0 @@
-import React, { FC, ReactNode } from 'react'
-import styled from 'styled-components'
-
-import { Text } from '../Text'
-import { theme } from '../theme'
-
-type Props = {
-  id: string
-  checked: boolean
-  children: ReactNode
-  toggle: () => void
-}
-
-export const CheckBox: FC<Props> = ({ id, checked, children, toggle }) => (
-  <BoxContainer id={id}>
-    <Text tag="span" typo="base">
```

Comment:
This component was only used for `CheckBoxGroup`, I've deleted to use the `CheckBox` component instead as it was a copy paste of the Checkbox component